doc/design.txt

```diff
@@ -272,7 +272,7 @@ purposes; they are not actually present in the boot vector.
     -----------------+--------+--------'
      swap: none                        |
     -----------------------------------'
-    
```

Comment:
> Right, "State I" and "State V" are really two states in the flash. But both of them end up being NONE in the code.

Well, they're two "vector states", which both lead to the same "swap state" or "swap result" ("vector state" being an input to the bootloader, "swap state" being an output or at least an intermediate output).

You bring up a good point in the earlier sections discussion that this isn't really a listing of states so much as it is a sequential listing of what the bootloader is checking. Perhaps that is a better way to write this section too?